WcaOnRails/app/controllers/results_controller.rb

```diff
@@ -29,49 +29,82 @@ def rankings
     @show = splitted_show_param[0].to_i
     @is_persons = splitted_show_param[1] == "persons"
     @is_results = splitted_show_param[1] == "results"
-    limit_condition = "LIMIT #{@show}"
+    expanded_limit_condition = "LIMIT #{@show * 2}"
```

Comment:
I guess this guarantees that everyone will be shown (except for weird edge cases where more than 2n people are tied?)

Reply:
This value can likely be a lot smaller (maybe the limit + 25 or somthing), as it's highly unlikely to run into a case where even 25 people are tied for 100th

Reply:
Fewest moves is probably going to be over 25 tied near the 100th spot at one point